admin/merge_pr.py

```diff
@@ -279,6 +285,26 @@ def transform(x):
     return None
 
 
+def jenkins_info_from_response(project):
+    """
+    Get the Jenkins job information from the Jenkins API response.
+
+    :param dict project: the API response as a dict.
+    :return (JenkinsResults, dict): The first element will
```

Comment:
Yeah, it's not that nice, I agree.

The reason it's like this is for that filter() that you mention, so that we can get the
information, then filter it on parts of that info, but use other parts.

Better schemes might be to build a dict of all the info, then filter the keys. The
functions that use the results can then just do a dict lookup for what they
need.

Alternatively I could create some object that would hold all the info with
names rather than positions, perhaps a namedtuple or a class.